evmcore/apply_fake_genesis.go

```diff
@@ -94,7 +94,18 @@ func MustApplyFakeGenesis(statedb *state.StateDB, time inter.Timestamp, balances
 
 // FakeKey gets n-th fake private key.
 func FakeKey(n uint32) *ecdsa.PrivateKey {
-	var keys = [100]string{
+	/* NOTE: hardcoded keys are from legacy (golang 1.17)
+	func FakeKey(n uint32) *ecdsa.PrivateKey {
+		reader := rand.New("math/rand".NewSource(int64(n)))
+		key, err := ecdsa.GenerateKey(crypto.S256(), reader)
+		if err != nil {
+			panic(err)
+		}
+		return key
+	}
+	*/
+	var keys = [101]string{
+		"0x41d3ff12045b73c870529fe44f70dca2745bafbe1698ffc3c8759eef3cfbaee1",
```

Comment:
Adding more item in the first place will generate incompatibility validator keys with previous version of fakenet I think